robotiq_2f_85_gripper_visualization/urdf/robotiq_arg2f_85_model_macro.xacro

```diff
@@ -153,7 +153,7 @@
       <parent link="${prefix}${fingerprefix}_outer_finger" />
       <child link="${prefix}${fingerprefix}_inner_finger" />
       <axis xyz="1 0 0" />
-      <limit lower="0" upper="0.8757" velocity="2.0" effort="1000" />
+      <limit lower="-0.8757" upper="0.8757" velocity="2.0" effort="1000" />

```

Comment:
I think `upper` should be `0`.
I think https://github.com/ros-industrial/robotiq/pull/119 is correct and https://github.com/ros-industrial/robotiq/pull/157 is insufficient (though this is not a practical problem because we usually do not feel inconvenience when mimic joint's range is larger than the original one).